build/scripts/build.ps1

```diff
@@ -540,10 +495,10 @@ function Deploy-VsixViaTool() {
     }
 
     Write-Host "Installing all Roslyn VSIX"
-    foreach ($e in $all) {
-        $name = Split-Path -leaf $e
-        $filePath = Join-Path $configDir $e
-        $fullArg = "$baseArgs $filePath"
+    $vsixFiles = Get-ChildItem -Path $vsSetupDir -Include "*.vsix"
```

Comment:
Part of the reason we used an explicit array before was to guarantee ordering. This new code essentially has a non-deterministic ordering and I believe can lead to installation isuses because dependencies aren't guaranteed to be installed in the right order.

CC @jasonmalinoski

Reply:
I see. I'll revert and a dd a comment.